docs/site/Decorators.md

```diff
@@ -132,56 +132,181 @@ accordingly or do a composition of them:
 
 ### Parameter Decorator
+ 
+  Syntax: see [API document](https://github.com/strongloop/loopback-next/blob/master/packages/openapi-v3/src/parameter-decorator.ts#L18-L29)
 
-  Syntax: [`@param(paramSpec: ParameterObject)`]()
+  Shortcut Pattern: `@param.${in}.${type}(${name})`
 
-  Optional Pattern: `@param.${in}.${type}(${name})`
-
-  `@param` can be applied to method itself or specific parameters, it describes
-  an input parameter of a Controller method.
+  `@param` is applied to method parameters to generate OpenAPI parameter specification for it.
 
   For example:
 
   ```ts
-    // example 1: decorator `@param ` applied on method level
-    class MyController {
-      @get('/')
-      @param(offsetSpec)
-      @param(pageSizeSpec)
-      list(offset?: number, pageSize?: number) {}
-    }
+  import {get, param} from '@loopback/rest';
+
+  const categorySpec = {
+    name: 'category',
+    in: 'path',
+    required: true,
+    schema: {type: 'string'}
+  }
+
+  const pageSizeSpec = {
+    name: 'pageSize',
+    in: 'query',
+    required: false,
+    schema: {type: 'integer', format: 'int32'}
+  }
+
+  class MyController {
+    @get('Pets/{category}')
+    list(
+      @param(categorySpec) category: string,
+      @param(pageSizeSpec) pageSize?: number,
+    ) {}
+  }
   ```
 
-  or
+  Writing the whole parameter specification is tedious, we create shortcuts to define params with pattern `@param.${in}.${type}(${name})`:
+
+  - in: the parameter location, one of the following values: `query`, `header`, `path`.
+  - type: a [common name of OpenAPI primitive data type](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#data-types), 
+    take specifying a data get from query as an example, a list of available shortcuts can
+    be found in [API document](http://apidocs.loopback.io/@loopback%2fopenapi-v3/#param.query).
+  - name: name of the parameter, should be a `string`, 
 
+  An equivalent example using shortcut decorator would be:
+  
   ```ts
-    // example 2: decorator `@param` applied on parameter level
     class MyController {
-      @get('/')
+      @get('/Pets/{category}')
       list(
-        @param(offsetSpec) offset?: number,
-        @param(pageSizeSpec) pageSize?: number,
+        @param.path.string('category') category: string,
+        @param.query.number('pageSizes') pageSize?: number,
       ) {}
     }
   ```
 
-  In the first example, we apply multiple decorators to a single declaration.
-  The order in which the decorators are called is important because a `@param`
-  decorator must be applied after an operation decorator.
-  To learn more about TypeScript decorator composition,
-  refer to [TypeScript Decorator Documentation](https://www.typescriptlang.org/docs/handbook/decorators.html)
+  You can find the specific usage in [Writing Controller methods](controller.htm#writing-controller-methods)
+
+  *We are supporting parameter location 'cookie' in the feature, track the feature in story*
```

Comment:
_The parameter location `cookie` is not supported yet, see https://github.com/strongloop/loopback-next/issues/997._